include/cutlass/gemm/collective/intel_pvc_mma.hpp

```diff
@@ -101,13 +101,14 @@ struct CollectiveMma<
   static constexpr int SubgroupSize = DispatchPolicy::SubgroupSize;
 
   using DpasShape = typename TiledMma::Shape_MNK;
-  using TileDpasShape = decltype(tile_shape(TiledMma()));
+  using SubgroupTileShape = decltype(tile_shape(TiledMma()));
 
-  static constexpr uint32_t MaxThreadsPerBlock = get<0>(DpasShape()) * get<1>(DpasShape());
+  static constexpr uint32_t MaxThreadsPerBlock =
+          cute::size(WorkgroupTileShape{}) / cute::size(SubgroupTileShape{})* SubgroupSize;
 
-  static constexpr int FragsM = get<0>(TileDpasShape{}) / get<0>(DpasShape()); // A frags per sub_group
-  static constexpr int FragsN = get<1>(TileDpasShape{}) / get<1>(DpasShape()); // B frags per sub_group
-  static constexpr int FragsK = get<2>(TileDpasShape{}) / get<2>(DpasShape());
+  static constexpr int FragsM = get<0>(SubgroupTileShape{}) / get<0>(DpasShape()); // A frags per sub_group
```

Comment:
Here also would it be possible to change the DpasShape -> AtomTileshape

Reply:
That could be the copy atom or the mma atom. I can use `MmaAtomShape` or `DpasAtomShape`  instead

Inside the `TiledMma` struct it is defined as `using AtomShape_MNK  = typename MMA_Atom::Shape_MNK;`

Reply:
Yes for MMA collective lets use mmaAtomShape and for copy part lets use copyAttomShape